src/middleware/stats.coffee

```diff
@@ -16,16 +16,34 @@ exports.incrementTransactionCount = (ctx, done) ->
     sdc.increment domain + '.Channels' # Overall Counter
     sdc.increment domain + '.Channels.' + transactionStatus #Overall Transaction Status
     sdc.increment domain + '.Channels.' + ctx.authorisedChannel._id # Per channel
-    sdc.increment domain + '.Channels.' + ctx.authorisedChannel._id + '.' + transactionStatus # Per Channel Status
+    sdc.increment domain + '.Channels.' + ctx.authorisedChannel._id + '.Statuses.' + transactionStatus # Per Channel Status
 
     #Collect stats for non-primary routes
     if ctx.routes?
       for route in ctx.routes
-        sdc.increment domain + '.Channels.' + ctx.authorisedChannel._id + '.' + route.name # Per non-primary route
-        sdc.increment domain + '.Channels.' + ctx.authorisedChannel._id + '.' + route.name + '.' + route.response.status # Per route response status
+        sdc.increment domain + '.Channels.' + ctx.authorisedChannel._id + '.nonPrimaryRoutes.' + route.name # Per non-primary route
+        sdc.increment domain + '.Channels.' + ctx.authorisedChannel._id + '.nonPrimaryRoutes.' + route.name + '.StatusCodes.' + route.response.status # Per route response status
+
+        if route.orchestrations?
+          for orchestration in route.orchestrations
+            orchestrationStatus = orchestration.response.status
+            sdc.increment domain + '.Channels.' + ctx.authorisedChannel._id + '.nonPrimaryRoutes.' + route.name + '.orchestrations.' + orchestration.name
+            sdc.increment domain + '.Channels.' + ctx.authorisedChannel._id + '.nonPrimaryRoutes.' + route.name + '.orchestrations.' + orchestration.name + '.StatusCodes.' + orchestrationStatus
+            sdc.increment domain + '.Channels.' + ctx.authorisedChannel._id + '.nonPrimaryRoutes.' + route.name + '.StatusCodes.' + route.response.status + '.orchestrations.' + orchestration.name
+            sdc.increment domain + '.Channels.' + ctx.authorisedChannel._id + '.nonPrimaryRoutes.' + route.name + '.StatusCodes.' + route.response.status + '.orchestrations.' + orchestration.name + '.StatusCodes.' + orchestrationStatus
```

Comment:
Minor comment: I see that sometime you are using a capital first letter in the namespace and sometime you aren't. I think we should standardise this to make it look better. Maybe we could rather just use a lowercase first letter everywhere instead.